### ISSUE TITLE
Renovate doesn't update title and PR description when updating the patch with an even newer version number

### ISSUE THREAD

**Issue by felixfbecker** [renovatebot/renovate#7717]
**What Renovate type, platform and version are you using?**
GitHub
**Describe the bug**

When a new version comes out for a package that already has a PR, Renovate updates the patch with the new version, but doesn't update the title, the PR description table or the release notes:

![image](https://user-images.githubusercontent.com/10532611/99013197-475fbc80-2550-11eb-9abf-4efe144cc557.png)

https://github.com/felixfbecker/svg-screenshots/pull/17/files

**Comment by rarkins** [comment]
Looks like it's updated now though?

**Comment by HonkingGoose** [comment]
@rarkins I'm not sure what to do with this issue, I could label it `bug` but it seems to have fixed itself already.
This might also be the type of bug that can only be fixed if we have a reproduction?

I'll let you decide on how to label the issue, or you can close it if you don't want to keep track of this anymore.

**Comment by felixfbecker** [comment]
It seems to be that there is always a brief period where the version is updated but the PR title and changelog is not. Which is somewhat dangerous because a maintainer might merge a change that they didn't intend to merge (because the changelog wasn't noting it).

**Comment by rarkins** [comment]
Yes, it's a bug, I think the PR title/body update is not triggered on the same run where the package and lock file is updated. Need to confirm if it affects lockfile-only updates too, or only this particular case where package file needs updating too.
